cmd/juju/action/do_test.go

```diff
@@ -366,8 +367,10 @@ func (s *DoSuite) TestRun(c *gc.C) {
 			restore := s.patchAPIClient(fakeClient)
 			defer restore()
 
-			s.subcommand = &action.DoCommand{}
-			ctx, err := testing.RunCommand(c, s.subcommand, t.withArgs...)
+			wrappedCommand, _ := action.NewDoCommand()
+			args := append([]string{"-e", "dummyenv"}, t.withArgs...)
+			c.Logf("XXX ARGS: %#v", args)
```

Comment:
I'm going to be opinionated and say these should be logged as NSFW ARGS instead.